src/index.html

```diff
@@ -8,6 +8,39 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Product cards</h1>
+    <div class="products__item card" data-qa="card">
+      <img src="images/imac.jpeg" alt="iMac 27"
+      class="card__img"
+      width="160"
+      height="134">
```

Comment:
некорректные переносы, высоту ширину через стили задавай
```suggestion
      <img
        src="images/imac.jpeg"
        alt="iMac 27"
        class="card__img"
      >
```